### PR TITLE
TI-93 Heroku custom vim plugin

### PR DESCRIPTION
**Pull request by MSiccardi**
### Test Plan:

**Use vim to open a file from Heroku CLI**
Setup: 
- Download code
- Set VIM_SOURCE env variable to static vim S3 url
-  From repository root run `heroku plugins:link`

Execution: Run `heroku vim -a <testing_app>`
Assertion: From console open file with vim, for example `vim /app/manage`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,56 @@
...
+<!-- toc -->
+* [Installation](#installation)
+* [Usage](#usage)
+* [Commands](#commands)
+<!-- tocstop -->
+
+# Installation
+<!-- installation -->
+```sh-session
+$ heroku plugins:install @diurnate/heroku-vim-plugin
```
Comment (copelco) [inline README.md:17]: Is this on Confluence anywhere yet? I looked briefly, like on the Deployment page, but didn't see it.

File: src/utils.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const {Dyno} = require('@heroku-cli/plugin-run-v5')
```
Comment (copelco) [inline src/utils.js:1]: Cool!

File: README.md [hunk 3]
```diff
@@ -1 +1,56 @@
...
+```
+<!-- installationstop -->
+
+# Usage
+<!-- usage -->
+```sh-session
+$ heroku vim -a <app>
```
Comment (copelco) [inline README.md:34]: It's not entirely clear to at which point ``VIM_SOURCE`` needs to exist within the environment. This fails locally for me if I don't have ``VIM_SOURCE`` defined. Perhaps add an example setting that above this (as well as to Confluence above)?

**Review by copelco — APPROVED** [review]
Worked for me with both small and large files on Heroku. I didn't test the ``plugins:install`` functionality yet since it doesn't work until we merge this branch.

Added some comments about documentation that I think should be addressed before merging this, but otherwise it look good to me :shipit:

File: README.md (hunk 3 above)
Comment (MSiccardi) [reply README.md:34]: Definitely, I updated the code to change the source but missed the README, I'll update it

File: README.md (hunk 1 above)
Comment (MSiccardi) [reply README.md:17]: Not yet, I can summarize the usage and add a link to the repository for the details, or should I add all the details in the confluence page as well?

Comment (copelco) [reply README.md:17]: I think Confluence too

File: README.md (hunk 3 above)
Comment (MSiccardi) [reply README.md:34]: Updated!

File: README.md (hunk 1 above)
Comment (MSiccardi) [reply README.md:17]: Just updated Confluence docs, Deployment page

**Review by emullaney — APPROVED** [review]
Reviewed the docs only and they look fine to me :book:

**Pull request merged**